app/models/candidate_interface/sign_up_form.rb

```diff
@@ -0,0 +1,27 @@
+module CandidateInterface
+  class SignUpForm
+    include ActiveModel::Model
+
+    attr_accessor :email_address, :accept_ts_and_cs
+    validates :email_address, :accept_ts_and_cs, presence: true
+    validates :email_address, length: { maximum: 250 },
+                              format: { with: Candidate::NOTIFY_EMAIL_REGEXP }
+
+
+    def save_base(candidate)
+      unless candidate.valid?
+        candidate.errors[:email_address].each{|error| errors.add(:email_address, error)}
```

Comment:
The error message we get here is:

![image](https://user-images.githubusercontent.com/42817036/68463521-be0ffe80-0206-11ea-87ea-e5115d9dc4a4.png)

We should add an error message like the one suggested in the GOV.UK Design System (see https://design-system.service.gov.uk/patterns/email-addresses/#error-messages). Possibly: `Enter an email address in the correct format, like name@example.com`.